### ISSUE TITLE
Screen close/open button bug

### ISSUE THREAD

**Issue by hpbitleaf** [rafaelvcaetano/melonDS-android#182]
* **Device model:** Pixel 2
* **Android version:** Android 9

**Problem:**  
The button that lets you "open/close" the DS has bug. In any game that supports closing screen, the second time using it in the same game causes it to stop working. I originally saw this issue posted by someone on reddit but I guess they couldn't be bothered to make a report.

**Steps to reproduce (if applicable):**
1. Loads any game that turns screens off when DS is closed.
2. Press screen close button, to "sleep" once, then tap off.
3. Do it again in the same game, same running instance, next time you try to "open" the DS it gets stuck on white screens forcing exit/reload.

**Comment by rafaelvcaetano** [comment]
Yep, I'm aware of that issue. However, this problem only happens when using a custom BIOS. So, while this issue is not fixed, you can use the internal BIOS as a workaround.
